src/main.rs

```diff
@@ -15,6 +15,7 @@ use std::thread::sleep;
 use std::time::{Duration, Instant, SystemTime, UNIX_EPOCH};
 use sysinfo::{CpuExt, ProcessExt, RefreshKind, System, SystemExt};
 
+// TODO: rename from french(?) "cunter" to english "counter"
```

Comment:
Didn't want to do this in this PR, but a replace all of "cunter" with "counter" would work.

Reply:
done!